bundles/org.openhab.binding.mikrotik/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,431 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mikrotik"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="routeros">
+		<label>Mikrotik RouterOS</label>
+		<description>A connection to RouterOS device</description>
+		<channels>
+			<channel id="freeSpace" typeId="freeSpace"/>
+			<channel id="totalSpace" typeId="totalSpace"/>
+			<channel id="usedSpace" typeId="usedSpace"/>
+			<channel id="freeMemory" typeId="freeMemory"/>
+			<channel id="totalMemory" typeId="totalMemory"/>
+			<channel id="usedMemory" typeId="usedMemory"/>
+			<channel id="cpuLoad" typeId="cpuLoad"/>
+			<channel id="uptime" typeId="uptime"/>
+			<channel id="upSince" typeId="upSince"/>
+		</channels>
+		<properties>
+			<property name="vendor">Mikrotik</property>
+			<property name="modelId">RouterOS</property>
+			<property name="firmware"/>
+			<property name="serial"/>
+		</properties>
+		<representation-property>name</representation-property>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Hostname</label>
+				<description>Hostname or IP address of the RouterOS device</description>
+				<default>192.168.88.1</default>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="integer" max="65535" min="1" required="false">
```

Comment:
```suggestion
			<parameter name="port" type="integer" max="65535" min="1" required="true">
```
I take it the whole binding will not work if there is no port defined?

Reply:
When port is not defined, a default of 8728 will be used.